onedal/basic_statistics/tests/test_incremental_basic_statistics.py

```diff
@@ -189,3 +189,60 @@ def test_all_option_on_random_data(
             gtr = function(data)
         tol = fp32tol if res.dtype == np.float32 else fp64tol
         assert_allclose(gtr, res, atol=tol)
+
+
+@pytest.mark.parametrize("queue", get_queues())
+@pytest.mark.parametrize("dtype", [np.float32, np.float64])
+def test_incremental_estimator_pickle(queue, dtype):
```

Comment:
Since this tests serialization, it should also test that changes in regular attributes like `batch_size` or `n_jobs` remain after deserializing.